### ISSUE TITLE
Using Old VS Code Calva startup fails with "command not found" for Fire up Getting Started REPL, or jack-in

### ISSUE THREAD

**Issue by gunnarx** [BetterThanTomorrow/calva#1584]
Issue Type: <b>Bug</b>

Steps to reproduce:

1) Install Calva extension
2) CTRL-P, select "Fire up Getting Started REPL"

(Similar results "command not found" happen for any other operation, such as jack-in)

Expected Result:
Getting started REPL should start and Calva should be active.

Actual result:

- Popup log message reads: "Command 'Calva: Fire up the ”Getting Started” REPL' resulted in an error (command 'calva.startStandaloneHelloRepl' not found)"
- Calva does not seem to "start"
- No getting-started buffers are opened.

Developer console reports two noteworthy messages:

- /usr/share/code/resources/app/out/vs/workbench/workbench.desktop.main.js:62 [Extension Host] (node:2180444) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.

- /usr/share/code/resources/app/out/vs/workbench/workbench.desktop.main.js:1904 Activating extension 'betterthantomorrow.calva' failed: Cannot read property 'createTestController' of undefined.

Extension version: 2.0.244
VS Code version: Code 1.55.2 (3c4e3df9e89829dce27b7b5c24508306b151f30d, 2021-04-13T09:37:02.931Z)
OS version: Linux x64 5.14.11-100.fc33.x86_64

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-9600K CPU @ 3.70GHz (6 x 1709)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: disabled_software<br>skia_renderer: enabled_on<br>video_decode: disabled_software<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 2, 2|
|Memory (System)|31.21GB (1.15GB free)|
|Process Argv|. --no-sandbox --crash-reporter-id 2c80c5f1-66f5-4848-b3d6-6fc54bad91a8|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|i3|
|XDG_CURRENT_DESKTOP|i3|
|XDG_SESSION_DESKTOP|i3|
|XDG_SESSION_TYPE|x11|
</details>

**Comment by PEZ** [comment]
Thanks for reporting! I am very sorry that Calva fails for you here. Please consider using the Getting Started REPL in the browser for now, and we will try fix the issue as fast as we can: https://calva.io/get-started-with-clojure/

**Comment by PEZ** [comment]
To help bisect this. Can you try with Calva v2.0.232? 

According to the changelog I think that is the last version before we started to initialize andy test-ui stuff at startup: https://github.com/BetterThanTomorrow/calva/blob/published/CHANGELOG.md#20232---2021-12-31

To use another version of Calva: Open the Extensions pane and click the cogwheel on the Calva item, select **Install Another Version...**

<img width="626" alt="image" src="https://user-images.githubusercontent.com/30010/157224352-2df91854-ace9-49bb-ba6d-d2fa7fe87df1.png">

**Comment by gunnarx** [comment]
I uninstalled the plugin and installed again.  Same thing happens now that it is on v2.0.252

OK, will try 2.0.232

**Comment by gunnarx** [comment]
Seems ~like the same~ to work with 2.0.232.   That was remarkably easy to do, so just give me a few minutes and I will continue bisecting, and confirm fully.

**Comment by gunnarx** [comment]
Yes, 2.0.232 seems to work and 2.0.233 is the first bad version.

**Comment by PEZ** [comment]
I now see that your VS Code version is `1.55.2`. Maybe that is the reason. Can you try with latest VS Code?

**Comment by gunnarx** [comment]
Updated VS Code.  Calva 2.0.252 works with VS code 1.65.1

**Comment by gunnarx** [comment]
So I suppose it's OK to just not fix anything and move on...

**Comment by PEZ** [comment]
Thanks for confirming!

Leaving this open because we need to address it somehow. Not sure how right now, but anyway.
